cortex-m-semihosting/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.4.1] - 2020-10-20
```

Comment:
What did we say we'd do about those links again? Maintain them with special tags? If not we can just give them up but then this shouldn't be a link.

Reply:
Thanks for catching that, I had intended for it to be a link and I'll make the tag upon release. I've pushed an update.